watertap/data/techno_economic/brine_concentrator.yaml

```diff
@@ -144,3 +160,166 @@ default:
       value: 0.98
       units: dimensionless
       constituent_longform: zinc
+
+crystallizer:
+  elec_coeff_1:
+    value: 56.7
+    units: kWh/m^3
+    reference: Survey of High-Recovery and Zero Liquid Discharge Technologies for Water Utilities (2008).
+  elec_coeff_2:
+    value: 1.83E-5
+    units: L/mg*kWh/m^3
+    reference: Survey of High-Recovery and Zero Liquid Discharge Technologies for Water Utilities (2008).
+  elec_coeff_3:
+    value: 9.47
+    units: kWh/m^3
+    reference: Survey of High-Recovery and Zero Liquid Discharge Technologies for Water Utilities (2008).
+  elec_coeff_4:
+    value: 8.63E-4
+    units: kWh/m^6*hr
+    reference: Survey of High-Recovery and Zero Liquid Discharge Technologies for Water Utilities (2008).
+  recovery_frac_mass_H2O:
+    value: 0.95
```

Comment:
@luohezhiming did you change this value? Nothing wrong with it if there is a reference, I just know water_recovery is 0.9 in WT3.

Reply:
I copy this water_recovery from the original crystallizer.yaml file. The water_recovery for a brine_concentrator is 0.9, but is given 0.95 in the original crystallizer.yaml file.

Reply:
Ah perfect, I missed that this was for the subtype